packages/@rnw-scripts/just-task/just-task.js

```diff
@@ -21,9 +21,14 @@ const {
   task,
   tscTask,
   tscWatchTask,
-  parallel
+  parallel,
+  addResolvePath,
 } = require('just-scripts');
 
+// Allow searching from current directory to be able to resolve peerDependencies
+// when running without hoisting
+addResolvePath(__dirname);
+
```

Comment:
I'm not 100% sure this would do the right thing in cases where different packages bring in different versions of the tooling. But this is more correct than before.

Reply:
From some experimentation I think it will resolve correctly to the version provided by the package which references it.